src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -231,6 +231,11 @@ export class OrganizeImportsAction extends EditorAction {
 				primary: KeyMod.Shift | KeyMod.Alt | KeyCode.KeyO,
 				weight: KeybindingWeight.EditorContrib
 			},
+			metadata: {
```

Comment:
Please remove the extra new lines and put the arguments to `localize` on a single line